shared/middleware/helpers/dynamo_table.rb

```diff
@@ -1,4 +1,5 @@
 require 'aws-sdk'
+require 'sinatra'
```

Comment:
`DynamoTable` depends on `Sinatra::NotFound`, but that dependency was only provided indirectly by the other requires in `tables_api.rb`. This is the first time anyone is requiring `dynamo_table.rb`directly outside of `tables_api.rb`. Similarly, `storage_id` has always had `base64` provided by whoever was requiring `storage_id` until now.